.github/workflows/ci.yml

```diff
@@ -40,3 +40,18 @@ jobs:
         TRAVIS: true # Pretend to be Travis. More info: https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/util/ci_provider.js
         TRAVIS_PULL_REQUEST_BRANCH: ${{ github.head_ref }} # Tell Cypress what the PR branch name is
         CYPRESS_RECORD_KEY: 3a9347b6-36ab-4a36-823d-709f4078b148
+
+	dependency_test:
```

Comment:
@anicholls I'm not sure how this happened, but these are tab characters (`\t`) instead of space characters (`\s`). I'm not sure why this isn't failing a yaml validation on Github's part, but the significant whitespace issue is preventing this workflow file from running. Github is conveniently highlighting in red.